### PR TITLE
DOC: Remove jquery, popper and boosted.js from imports

### PR DESCRIPTION
**Pull request by Lausselloic**
remove jquery, popper, and boosted.js from documentation and replace all demo using data-bs-* element by ng-bootstrap components

### CODE REVIEW

File: src/app/demos/scrollspy.component.html [hunk 1]
```diff
@@ -9,18 +9,18 @@
...
-      <ul class="dropdown-menu">
-        <li><a class="dropdown-item" (click)="onClick('scrollspyHeading3')">Third</a></li>
-        <li><a class="dropdown-item" (click)="onClick('scrollspyHeading4')">Fourth</a></li>
-        <li><hr class="dropdown-divider"></li>
-        <li><a class="dropdown-item" (click)="onClick('scrollspyHeading5')">Fifth</a></li>
-      </ul>
+    <li ngbDropdown class="nav-item">
+      <a href (click)="false" class="nav-link" ngbDropdownToggle>Dropdown</a>
+      <div ngbDropdownMenu>
+        <button ngbDropdownItem (click)="onClick('scrollspyHeading5')">Third</button>
```
Comment (julien-deramond) [inline src/app/demos/scrollspy.component.html:15]: `(click)="onClick('scrollspyHeading3')"` (and 4 for the next one)

File: .travis.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ sudo: false
 
 language: node_js
 node_js:
-  - "12.14.0"
+  - "14.15.5"
```
Comment (julien-deramond) [inline .travis.yml:6]: I would use the latest version of Node: 14.18.1

**Review by julien-deramond — CHANGES_REQUESTED** [review]
Haven't reviewed is entirely yet but here are some first comments:
* Got some errors after a `npm run lint`
* `package-lock.json` is modified when I run `npm i` with Node v14.18.1 and NPM 6.14.15

File: src/app/demos/scrollspy.component.html (hunk 1 above)
Comment (julien-deramond) [inline src/app/demos/scrollspy.component.html:15]: `(click)="onClick('scrollspyHeading3')"` (and 4 for the next one)

**Review by julien-deramond — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (Lausselloic) [reply .travis.yml]: updated

File: src/app/demos/scrollspy.component.html (hunk 1 above)
Comment (Lausselloic) [reply src/app/demos/scrollspy.component.html]: fixed

File: src/app/demos/modal.component.html [hunk 3]
```diff
@@ -1,21 +1,17 @@
...
         <h5 class="modal-title" id="exampleModalLabel">Modal title</h5>
-        <button type="button" class="btn-close" data-bs-dismiss="modal"><span class="visually-hidden">Close</span></button>
+        <button type="button" class="btn-close" (click)="modal.dismiss('Cross click')"><span class="visually-hidden">Close</span></button>
     </div>
     <div class="modal-body">
         ...
     </div>
     <div class="modal-footer">
-        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
+        <button type="button" class="btn btn-secondary" (click)="modal.dismiss('Save click')">Close</button>
```
Comment (julien-deramond) [inline src/app/demos/modal.component.html:10]: It is rather 'Cross click' as well here.

File: src/app/demos/swiper.component.ts [hunk 4]
```diff
@@ -2,12 +2,6 @@ import { Component } from '@angular/core';
 
 @Component({
     selector: 'demo-swiper',
-    styles: [`
```
Comment (julien-deramond) [inline src/app/demos/swiper.component.ts:5]: Cannot test it so I trust you on this 🙈

File: src/app/demos/tabs.component.html [hunk 5]
```diff
@@ -45,33 +45,32 @@ <h6>Vertical nav</h6>
...
 <h6>Tabs</h6>
-
-<ul class="nav nav-tabs" id="myTab" role="tablist">
-  <li class="nav-item" role="presentation">
-    <button class="nav-link active" id="home-tab" data-bs-toggle="tab" data-bs-target="#home" type="button" role="tab" aria-controls="home" aria-selected="true">Home</button>
+<ul ngbNav #nav="ngbNav" [(activeId)]="active" class="nav-tabs">
+  <li [ngbNavItem]="1">
+    <a ngbNavLink>Home</a>
+    <ng-template ngbNavContent>
+      <p>Raw denim you probably haven't heard of them jean shorts Austin. Nesciunt tofu stumptown aliqua, retro synth
```
Comment (julien-deramond) [inline src/app/demos/tabs.component.html:52]: I would have had a `.mb-0` to avoid the extra bottom margin (3 tabs)

**Review by julien-deramond — CHANGES_REQUESTED** [review]

File: src/app/demos/tabs.component.html (hunk 5 above)
Comment (Lausselloic) [reply src/app/demos/tabs.component.html:52]: I'll remove paragraph

**Review by julien-deramond — APPROVED** [review]

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**